src/Module/Create.php

```
@@ -94,7 +99,7 @@ private function createDirectoryStructure(string $modulePath, string $moduleName
             ));
         }
 
-        if (! mkdir($modulePath)) {
+        if (! mkdir($modulePath, 0777, true)) {
```

Will this be a security issue? I think, we should throw an exception only about file permission?

---

@hkulekci These are the default permissions when calling `mkdir()`; they're used here simply because we're also passing the third argument, the flag to recursively create the path.